### DISCUSSION TITLE
having trouble using the web interface

### DISCUSSION THREAD

**Discussion by AtariBaby** [airdcpp/airdcpp-windows#90] [General]
Is there a trick beyond the usual url ip address colon port? I can't get it to work: opened ports on my router, and turned off windows firewall, but i can't access the web interface. Is there something I'm missing?

4.20 windows version

**Comment by maksis** [comment]
- Are you trying to access it from the same computer? 
- What's the IP address that you are using? 
- What's the error that you see in the browser? 
- Are extensions working correctly?

**Comment by AtariBaby** [comment]
Hi, I’m trying to use it on the lan and on the wan. It will work if I use it on the computer it’s installed on.In most cases, my other apps like  sabnzbd and the “arrs” worked fine this way. I just entered the IP address and a colon and a port.  But I can’t get the air dc web version to work this way. I’m not sure what you mean by extensions. Do I need to install extensions?GregorySent from my iPhoneOn Jun 20, 2023, at 12:42 PM, maksis ***@***.***> wrote:﻿

Are you trying to access it from the same computer?
What's the IP address that you are using?
What's the error that you see in the browser?
Are extensions working correctly?

**Reply by Sopor** [reply]
Is this still an issue or have you manage to solved it @AtariBaby ?

**Comment by maksis** [comment]
There's no need to enter any IP address, just go to Settings -> Web server and click the link in the status text box
